### ISSUE TITLE
Internationalization

### ISSUE THREAD

**Issue by drogel** [meodai/color-names#81]
Hey!

Thanks for this awesome project, I can tell there is a huge amount of work behind it!

I was wondering, are there any plans for translating the existing color name list to other languages? In that case, I can help you with the spanish translations! :-D

**Comment by meodai** [comment]
thanks for your comment. 

There are no plans for translating the list, since lots of names are word plays it will be difficult sometimes impossible to translate them. I would suggest that you fork this repository and just create a spanish standalone version. I would be happy to link it in the readme and even add other languages to the API once you have a bunch of them. Let me know if you thing this is a good idea.

**Comment by drogel** [comment]
Indeed, I am not sure if all the names can be translated... that is why I think the list would have to be translated by native speakers, who can twist the wordplays when necessary to keep the core idea behind the name when translated.

For example, one of my favourites: April Fool's Red, would have to be "Rojo del Día de los Inocentes" in spanish. The "Día de los Inocentes" is the closest thing to April Fool's we've got in Spain, although it's not the same day, and they don't share the same history.

Anyways, I think your suggestion is a good idea for this case, I can start translating them on a forked project, and see how it goes. It will take a while because the list is huuuuuge! :-) But I will be happy to share the translations once I've got something, and then we can see what to do with it :-) Thanks!
